### ISSUE TITLE
Make text interpolation smooth

### ISSUE THREAD

**Issue by clabe45** [etro-js/etro#17]
As seen in *examples/introduction/keyframes.html*, text interpolation looks very choppy currently. I suspect this has to do with some font size rounding done by the browser. **TODO: Investigate**

**Comment by ChrisSMendoza** [comment]
Hey @clabe45, I'm looking to contribute to open source for Hacktoberfest and found this repo on **up-for-grabs**. When you say 'choppy' do you mean the shaking effect as it the text resizes? Or antialiasing? Or both?

**Comment by clabe45** [comment]
Hey, thanks for checking out my project! What I mean is the shaking effect as the text resizes. I haven't researched it extensively, but I suspect the reason this is happening is the canvas rendering context is rounding the font size.  

Let me know if you have any other questions!

**Comment by ChrisSMendoza** [comment]
Yeah of course! I'll keep looking into it and keep in mind the rounding of the font size. Sounds good!

**Comment by clabe45** [comment]
Hello, are you still interested in this issue? If not, I can unassign it so someone else can tackle it!
